frontend/src/views/Blacksmith.vue

```diff
@@ -261,6 +303,25 @@ export default {
       this.$refs['forge-details-modal'].show();
     },
 
+    onClickForge(i) {
+      this.chlickedForgeButton = i;
+      this.$refs['forge-element-selector-modal'].show();
+    },
+
+    setChoosenElement(ele, i) {
+      this.selectedElement = i;
+      if (i === 4) {
+        this.forgeCostMultiplier = 1;
+      } else{
+        this.forgeCostMultiplier = 2;
+      }
+      [].forEach.call(ele.srcElement.parentNode.childNodes, function(child) {
```

Comment:
don't do this. cast it to an array so you don't have to call it like this. also, use arrow functions.